patterns/2-structured/misusing metrics.md

```diff
@@ -0,0 +1,48 @@
+## Title
+
+Misusing Metrics
+
+## Patlet
+
+Metrics can be misused causing friction across teams. In these situations, create a sense of ownership within teams on how to use the data, avoiding comparisons, and begin measuring progress with deltas.
+
+## Problem
+
+The existence of metrics may lead to undesired situations and frictions across teams. Gamifications, list of rock stars or any other way of having people or teams “better” than others usually brings into the discussion defensive behaviors where the metric needs to be deprecated.
+
+## Context
+
+There is often a competitive spirit within development teams. In these situations, metrics tend to be used as evidence that supports a “better than” culture. This culture is a barrier to the healthy use of metrics to inform the successful progress in development velocity.
+
+## Forces
+
+* Individuals or teams may not understand the purpose of the metrics being used
+* Individuals or teams may feel threatened by how the metrics are being used
+* Leadership doesn’t understand how to utilize the metrics in a healthy way
+
+## Solutions
+
+Get a team internal agreement about the metrics that make sense in that specific team. Build a sense of ownership so everyone has contributed to the definition of the metric. Educate people about the right use of metrics and avoid comparisons. Measure deltas instead of net numbers, and look for outliers and how far projects are from the average instead of stating some behavior is “good” or “bad”.
```

Comment:
> Get a team internal agreement about the metrics that make sense in that specific team.

I am confused here. Are we talking about a metric that is only measured in a single team?
Or a metric that applies to a whole org, and is measured for each team belonging into that org?

Reply:
Adoption of metrics should start on the team level. First educate people about the best use of metrics i.g. avoiding comparisons, measure deltas instead of net numbers, look for outliers, estimate progress from the average or beginning, do not use "good" or "bad" as descriptions for individual or team performance. By building a common understand, everyone develops a sense of ownership because they contributed to the definition of the metric. Understand and utilization of a common metrics language starts with the individual then builds out to teams and finally entire organizations.